.github/workflows/autorelease.yml

```diff
@@ -23,11 +23,33 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.15
+      -
+        name: Import GPG signing key
+        uses: crazy-max/ghaction-import-gpg@v3
+        with:
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.PASSPHRASE }}
+      -
+        name: Debug
+        run:  |
+          echo "GPG ---------------------"
+          echo "fingerprint: ${{ steps.import_gpg.outputs.fingerprint }}"
+          echo "keyid:       ${{ steps.import_gpg.outputs.keyid }}"
+          echo "name:        ${{ steps.import_gpg.outputs.name }}"
+          echo "email:       ${{ steps.import_gpg.outputs.email }}"
+          echo "Go env ------------------"
+          pwd
+          echo ${HOME}
+          echo ${GITHUB_WORKSPACE}
+          echo ${GOPATH}
+          echo ${GOROOT}
+          env
       -
         name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v2
         with:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The `secrets.GITHUB_TOKEN` is an auto-generated token, no? Why change it?

Reply:
See https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Reply:
I guess we can re-use that.